### ISSUE TITLE
TGUI CRASH ON PANDEMIC INTRODUCING BEESEASE/PIERROT/BRAINROT/SEVERE ANXIETY/MAGNITIS

### ISSUE THREAD

**Issue by JillianLaville** [tgstation/tgstation#63570]
Reporting client version: 514.1571

## Round ID:
[174958](https://scrubby.melonmesa.com/round/174958)

TypeError: Cannot find the property 'length' of reference null or not defined
   at f (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:1125588)
   at Ie (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:179474)
   at Anonymous function (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:179742)
   at Le (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:179720)
   at Te (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:180573)
   at Le (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:179584)
   at Anonymous function (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:179742)
   at Le (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:179720)
   at De (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:180768)
   at Te (http://127.0.0.1:52094/tmp3692/tgui.bundle.js:1:180607)
User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0E; .NET4.0C; .NET CLR 3.5.30729; .NET CLR 2.0.50727; .NET CLR 3.0.30729; Tablet PC 2.0)
State: {"ckey":"iguanadie","interface":"Pandemic","window":{"key":"[0x201d18b]-main","size":null,"fancy":1,"locked":0}}

## Reproduction:

I got a virus crate from cargo,I put a beesease inside the pandemic machine which I built on another z-sector, beesease came from the cargo crate which I got from express delivery console, got this error, had to remove the culture via alt-click.
Similar errors comes when putting magnitis, gbs,pierrot,brainrot and severe anxiety.
Had to translate the first line of the error report over to english.
Seems to be about an error of special predefined diseases, as it took the normal rhinovirus culture without failing.

**Comment by SuperNovaa41** [comment]
This should have been fixed https://github.com/tgstation/tgstation/pull/63448

**Comment by jlsnow301** [comment]
I tested this on live after #63448 was merged and it was working. That looks like the previous error, also. 
If you can reproduce this on current, let me know and I'll make another hotfix

**Comment by TheBonded** [comment]
It's possible that a keyholder hasn't updated the servers, due to the feature freeze giving them no reason to urgently update them each time a fix pr is merged.

**Comment by timothymtorres** [comment]
> It's possible that a keyholder hasn't updated the servers, due to the feature freeze giving them no reason to urgently update them each time a fix pr is merged.

This was merged 5 days ago into the master branch.  I do believe that the servers get auto updated at least once a day if not more.   

> This should have been fixed #63448

Going to reopen the original issue then and close this one.  Thanks for the report @jlsnow301

**Comment by timothymtorres** [comment]
Duplicate of #63155

**Comment by TheBonded** [comment]
> 



> > It's possible that a keyholder hasn't updated the servers, due to the feature freeze giving them no reason to urgently update them each time a fix pr is merged.
> 
> This was merged 5 days ago into the master branch. I do believe that the servers get auto updated at least once a day if not more.

They are not, I've had to ask mothblocks to update the servers once or twice in the past when they hadn't been updated for a few days

**Comment by timothymtorres** [comment]
> They are not, I've had to ask mothblocks to update the servers once or twice in the past when they hadn't been updated for a few days

Wow you nailed it.  I just checked the commit history for that round and the master branch was outdated by almost a full week.  The PR to fix it would not have been present.  Since @jlsnow301 said he tested it on local with no problems I'm going to assume the fix worked properly and going to close the other issue report as well.

**Comment by JillianLaville** [comment]
Oops, haha, been playing campbell for a week and didn't see the changelog for new fixes, since it doesn't display them. Sorry dudes, the issue is long shift related then.
